src/com/facebook/buck/jvm/kotlin/KotlincToJarStepFactory.java

```diff
@@ -223,11 +220,8 @@ public void createCompileStep(
               kotlinc,
               ImmutableList.<String>builder()
                   .addAll(extraKotlincArguments)
-                  .add(NO_STDLIB)
-                  .add(NO_REFLECT)
                   .add(COMPILER_BUILTINS)
                   .add(LOAD_BUILTINS_FROM)
```

Comment:
Also, the `condenseCompilerArguments` would remove duplicates for this if user also added them :)

Reply:
True, I can move these as well. I admittedly am not familiar with these args

Reply:
I think it was me (or @cwoodwar6 ) who added them in order to get kapt to work. This was actually a copy past from what gradle outputs when doing the same.
Also, could you also remove them from [here](https://github.com/facebook/buck/blob/master/src/com/facebook/buck/jvm/kotlin/KotlincToJarStepFactory.java#L359-L360) and [here](https://github.com/facebook/buck/blob/master/src/com/facebook/buck/jvm/kotlin/KotlincToJarStepFactory.java#L387-L388) when adding them to `KotlinConfiguredCompilerFactory`?

Reply:
Consolidated in b48992c

Reply:
It actually looks like these two are false by default and not exactly documented. I'm not sure if they're maybe just required for kapt, so going to leave them as-is for now rather than move to the standard options. If they're not required for kapt either, we can just look at getting rid of them